chart/README.md

```diff
@@ -540,6 +541,29 @@ Some example commands are shown below.
    .
    ```
 
+#### Installing from remote repository
```

Comment:
Isn't this section duplicate ? or isn't it more relevant at the top of the readme (around the "Run" section) ?

Reply:
> Isn't this section duplicate ? or isn't it more relevant at the top of the readme (around the "Run" section) ?

If we change all the examples to install from the remote repo (which I think makes sense because that would now match our general instructions) we shouldn't need this example i.e.
- Remove this example
- Change all 
```
helm install 
... 
.
``` 
to 
```
helm repo add sonatype https://sonatype.github.io/helm3-charts/
helm install --namespace staging mycluster --dependency-update
...
sonatype/nexus-iq-server-ha
```

Reply:
Agree. I removed the example, since it is implicit in the others.
d973f803d777cd824da5a91b07c48039941ce268